listenbrainz/webserver/templates/index/privacy-policy.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'base.html' %}
+
+{% block title %}Privacy Policy - ListenBrainz{% endblock %}
+
+{% block content %}
+  <h2 class="page-title">ListenBrainz Privacy Policy</h2>
+
+  <h3>Important note</h3>
+
+  <p>
+    This privacy policy only addresses situations specific to ListenBrainz that are not covered
+    by the MetaBrainz organization wide privacy policy. <b>We strongly urge you to read the
+    full <a href="https://metabrainz.org/privacy">MetaBrainz privacy policy</a> before you
+    read the remainder of this ListenBrainz specific privacy policy.</b>
+  </p>
+
+  <h3>Use of your YouTube profile information</h3>
+
+  <p>
+    The embedded music player that is present in a number of pages on ListenBrainz can
+    <a href="{{ url_for('profile.music_services_details') }}">optionally be connected to
+    play YouTube content</a>. During this connection process ListenBrainz will request
+    permissions to view your YouTube profile data. Please note that YouTube does not have
+    any permissions lesser than this permission; if we wish to play YouTube content on
+    ListenBrainz pages, we are required to ask for this permission, even though we wish we
+    didn't have to do this.
```

Comment:
We can probably improve the wording of these lines. maybe @reosarevok has some suggestions ?

Reply:
I dont see anything wrong with it -- what do you feel is amiss?

Reply:
"Please note that YouTube does not have any permission options that are less wide than this one" maybe?

Reply:
How about this?
"Please note that in order to search for YouTube content to play on ListenBrainz pages, we have no choice but to ask for this permission.
We only use the absolute minimum data required to play music through Youtube (authentication to your Youtube account), and do not use it for any other purpose."

Reply:
> The embedded music player that is present in a number of pages on ListenBrainz can optionally be connected to    YouTube in order to play content. During the connection process ListenBrainz will request permissions to view your YouTube profile data. Unfortunately this is the minimum permission level offered by YouTube and thus if we wish to offer the ability to play YouTube content on ListenBrainz pages, we are required to ask for this permission, even though we do not need or want access to your YouTube profile. In summary, your data is used only to authenticate to your YouTube account and for no other purposes.

Reply:
I'll accept you suggestion with a minor edit. Thanks @atj !